ffi/pic.lua

```diff
@@ -170,7 +171,7 @@ function Pic.openGIFDocument(filename)
         error(string.format("Cannot parse GIF file: %s",
             ffi.string(giflib.GifErrorString(giffile.Error))))
     end
-    return GifDocument:new{giffile = giffile}
+    return GifDocument:new{giffile = giffile, err = err}
```

Comment:
Would just add `-- luacheck: no unused` , as GifDocument:new does not care.
I think it does not need to be the same err in both functions, it's just a receptacle for some error code.